packages/block-editor/src/components/use-settings/test/index.js

```diff
@@ -38,7 +44,19 @@ const mockCurrentBlockContext = (
 	);
 };
 
-describe( 'useSetting', () => {
+function runHook( hookCb ) {
+	let storedResult;
+	function TestHook() {
+		const result = hookCb();
+		useEffect( () => {
+			storedResult = result;
+		}, [ result ] );
+	}
+	render( <TestHook /> );
+	return storedResult;
+}
+
+describe( 'useSettings', () => {
```

Comment:
I'd keep a test for `useSetting()` too, even if it's just a single one.

Reply:
Added a test for `useSetting` :+1: